package.json

```diff
@@ -24,6 +24,7 @@
   "devDependencies": {
     "standard": "^17.0.0",
     "tap": "^16.3.0",
```

Comment:
Can we savely remove tap?

Reply:
Yes, we can. i keept it as test runner: it reads the tap output by tape and checks the coverage. I thought it is a a "standard" across the org and a friendly output